modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationRegistry.java

```diff
@@ -225,6 +252,10 @@ private CompletableFuture<Void> combineFutures(Collection<CompletableFuture<?>>
         };
     }
 
+    public Collection<Pattern> deletedPrefixes() {
```

Comment:
I wander if we should use a designated interface instead of `Collection<Pattern>`. Maybe in the future, I don't want to bother you anymore